packages/build/src/polymer-build.ts

```diff
@@ -11,3 +12,5 @@ export {getOptimizeStreams, OptimizeOptions} from './optimize-streams';
 export {PolymerProject} from './polymer-project';
 export {addServiceWorker, generateServiceWorker} from './service-worker';
 export {AddServiceWorkerOptions} from './service-worker';
+export {addWorkboxServiceWorker, generateWorkboxServiceWorker} from './service-worker-workbox';
+export {AddServiceWorkerWorkboxOptions} from './service-worker-workbox';
```

Comment:
@prateekbh Small nitpick. There is `addWorkboxServiceWorker` then `AddServiceWorkerWorkboxOptions`. Can you standardize to either having "Workbox" before or after "ServiceWorker"? It reads better if the placement of the additional word is consistent.

Reply:
Thanks. Will do this by EOD

Reply:
@prateekbh Ping.

Reply:
Sry! got caught up! doing it now

Reply:
@moderndeveloperllc done